CMakeLists.txt

```diff
@@ -71,6 +71,10 @@ if ("${K4A_ENABLE_LEAK_DETECTION_CMAKE}" STREQUAL "1")
     add_definitions(-DK4A_ENABLE_LEAK_DETECTION)
 endif()
 
+#
+set(CMAKE_BUILD_WITH_INSTALL_RPATH YES)
+set(CMAKE_INSTALL_RPATH "\$ORIGIN")
```

Comment:
>\$ORIGIN [](start = 25, length = 8)

Origin appear to be a token, so does that mean '\' is unnecessary? Example I see online are $ORIGIN\ which makes me think '\' is not needed.

Reply:
The quotes are there because its a complicated string. Without the quotes, it was unclear to me if the "\\" is there for CMake reasons or not. The quotes help me understand, this full string is passed to the compiler. The backslash is needed to escape the "$" in the command line (see this mailing list for more info https://cmake.org/pipermail/cmake/2008-January/019290.html).